llvm/lib/Target/NVPTX/NVPTXInstrInfo.td

```diff
@@ -747,6 +754,13 @@ def SELP_f16x2rr :
               [(set Int32Regs:$dst,
                     (select Int1Regs:$p, (v2f16 Int32Regs:$a), (v2f16 Int32Regs:$b)))]>;
 
+def SELP_i16x2rr :
```

Comment:
Needs sm90/ptx80 predicates?

Reply:
no this is supported on all platforms as it becomes just a int32 select.

Reply:
In that case, can we just use the existing `SELP_PATTERN`?
While we're at that, do the same for fp16/bf16.

Reply:
Good point, changed it to a pattern for all the cases